File: MATLAB/dir_NeuroElectro/NeuroElectro.m
function NeuroElectro()

    addpath(genpath('lib'));

    dataFileName = select_data_file();
    if strcmp(dataFileName, '!')
        return;
    end
    
    disp(' ');
    [header, data] = initialize_variables(dataFileName);

%    [parsedData] = parse_data(savedData, n);
    
    save_data(header, data, dataFileName);
    
    disp(' ');
    disp('Finis.');
    disp(' ');
end % NeuroElectro()

File: Python/dir_NeuroElectro/NeuroElectro.py
import pandas as pd

from lib.select_file import select_data_file
from lib.initialize import initialize_variables
from lib.save_data import save_data_file

dataFileName = select_data_file();
if dataFileName == '!':
    exit()
    
dataFrame = initialize_variables(dataFileName)

save_data_file(dataFrame, dataFileName)

print '\nFinis.\n'
